onnxruntime/core/optimizer/graph_transformer_utils.cc

```diff
@@ -257,12 +258,19 @@ InlinedVector<std::unique_ptr<GraphTransformer>> GenerateTransformers(
 
     case TransformerLevel::Level3: {
 #ifndef DISABLE_CONTRIB_OPS
+      const InlinedHashSet<std::string_view> cpu_ep = {onnxruntime::kCpuExecutionProvider};
       // Register the NCHWc layout transformer if supported by the platform.
```

Comment:
nit: could just move the cpu_ep in the Level2 case to outside the `switch` instead of declaring it here as well